web-console/5-components/quota-warnings.md

```diff
@@ -0,0 +1,58 @@
+# Quota Warnings
+
+## General
+- Distinguish between **compute-resource** limits and **object-count** quotas.
+- Both types of quotas/limits should be further broken down to specify the resource in question, e.g. Pods, Services, PVCs, Routes, Secrets, CPU, Memory.
+- Messages should be customizable.
+
+![quota kebab](img/quota-kebab.png)
+
+- Toasts and notification drawer messages should include a kebab menu with options to:
+  - View Quotas
+  - Don't Show Me Again
+  - Clear notification
+
+### Don't Show Me Again option
+- Option should be specific to each quota/limit.
+- Reset if the quota/limit changes. I.e. if I purchase additional capacity, I should get notifications if I reach/exceed the new quota.
+
+
+## On the Overview
+
+### At quota for object-count
+- User should be alerted via an informational message.
+- If the user takes an action that brings her to quota, that informational message should be toasted.
+- If the user logs into the system and it is at quota, a message should appear in the [Notification drawer](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/notifications) with no toast notification.
+- The message should include
+  - The resource in question
+  - The total used and the max allowed.
+  - Information/link to help the user take action if action is needed.
+  - Link to quota page.
+- The message should be customizable.  
+- For example: "You are using 1 of 1 Persistent Volume Claims. Contact your system administrator if you need additional resources. View Quotas"
+
+### At quota for compute-resource
+- User should be alerted via an informational message.
+- If the user takes an action that brings her to the quota, she should see a toast notification.
```

Comment:
This is really hard to anticipate. I'm not sure if we'll be able to know always.